packages/ra-materialui-treeview/README.md

```diff
@@ -0,0 +1,96 @@
+# ra-materialui-treeview
+
+A treeview component with [material-ui](https://github.com/mui-org/material-ui) to use with [react-admin](https://github.com/marmelab/react-admin).
+
+## Installation
+
+```sh
+npm install --save ra-materialui-treeview
+# or
+yarn add ra-materialui-treeview
+```
+
+## Dependencies
+
+If you want to enable [edition](#editable) (with drag and drop), ensure you also installed [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd)
+
+```sh
+npm install --save react-beautiful-dnd
+# or
+yarn add react-beautiful-dnd
+```
+
+## Usage
+
+With a category ressource having this shape:
```

Comment:
"this shape": be more precise. We're talking about a tree structure, where the parent-child relationship is stored using a `parent_id` field.